runtime/runtime-params-estimator/test-contract/src/lib.rs

```diff
@@ -47,6 +47,8 @@ extern "C" {
     fn sha256(value_len: u64, value_ptr: u64, register_id: u64);
     fn keccak256(value_len: u64, value_ptr: u64, register_id: u64);
     fn keccak512(value_len: u64, value_ptr: u64, register_id: u64);
+    fn ripemd160(value_len: u64, value_ptr: u64, register_id: u64);
+    fn ecrecover(hash_ptr: u64, v: u32, r_ptr: u64, s_ptr: u64, register_id: u64);
```

Comment:
This signature does not match the one in logic.rs

Reply:
Fixed: https://github.com/near/nearcore/pull/4380/commits/8c9341fe89506c030d56a02dafe8fae62e6eecbd